i18n/translations/ja-JP.json

```diff
@@ -0,0 +1,295 @@
+{
+    "% of the amount.": "% の金額.",
+    "%d of %d multisig": "%d の%d マルチシグネチャ",
+    "%d to %d": "%d へ %d",
+    "%s removed.": "削除されました.",
```

Comment:
Is this translation missing a "%s" for a replacement string?

Reply:
You're right, i don't know if i can change it here, but i think @odudex  can edit it before the final version! Thx so much for review it.

Reply:
@BitCoisas If you update your fork with the correction it will automatically update this pull request. I would be thankful if you could do that, just to ensure we won't forget that later.

Reply:
woahhhh, i didnt know that, thanks guys!

Reply:
Did it!